src/main/java/com/minecolonies/commands/ListCitizens.java

```diff
@@ -0,0 +1,171 @@
+        final int halfPage = (citizenCount % CITIZENS_ON_PAGE == 0) ? 0 : 1;
+        final int pageCount = ((citizenCount) / CITIZENS_ON_PAGE) + halfPage;
+
+        if (page < 1 || page > pageCount)
+        {
+            page = 1;
+        }
+
+        final int pageStartIndex = CITIZENS_ON_PAGE * (page - 1);
+        final int pageStopIndex = Math.min(CITIZENS_ON_PAGE * page, citizenCount);
+        final int prevPage = Math.max(0, page - 1);
+        final int nextPage = Math.min(page + 1, (citizenCount / CITIZENS_ON_PAGE) + halfPage);
+
+        List<CitizenData> citizensPage;
+
+        if (pageStartIndex < 0 || pageStartIndex >= citizenCount)
+        {
+            citizensPage = new ArrayList<>();
+        }
+        else
+        {
+            citizensPage = citizens.subList(pageStartIndex, pageStopIndex);
+        }
+
+        final ITextComponent headerLine = new TextComponentString("§2   ------------------ page " + page + " of " + pageCount + " ------------------");
+        sender.addChatMessage(headerLine);
+
+        for (final CitizenData citizen : citizensPage)
+        {
+            sender.addChatMessage(new TextComponentString(ID_TEXT + citizen.getId() + NAME_TEXT + citizen.getName()));
+
+            if(citizen.getCitizenEntity() != null)
+            {
+                final BlockPos position = citizen.getCitizenEntity().getPosition();
+                sender.addChatMessage(new TextComponentString(COORDINATES_TEXT + String.format("§4x=§f%s §4y=§f%s §4z=§f%s", position.getX(), position.getY(), position.getZ())));
+            }
+        }
+
+        final ITextComponent prevButton = new TextComponentString(" <- prev").setStyle(new Style().setBold(true).setColor(TextFormatting.GOLD).setClickEvent(
+          new ClickEvent(ClickEvent.Action.RUN_COMMAND, LIST_COMMAND_SUGGESTED+prevPage)
+        ));
+        final ITextComponent nextButton = new TextComponentString("next -> ").setStyle(new Style().setBold(true).setColor(TextFormatting.GOLD).setClickEvent(
+          new ClickEvent(ClickEvent.Action.RUN_COMMAND, LIST_COMMAND_SUGGESTED + + colonyId + " " + nextPage)
+        ));
+
+        final ITextComponent beginLine = new TextComponentString("§2 ----------------");
+        final ITextComponent endLine = new TextComponentString("§2---------------- ");
+        sender.addChatMessage(beginLine.appendSibling(prevButton).appendSibling(new TextComponentString("§2 | ")).appendSibling(nextButton).appendSibling(endLine));
+    }
+
+
+    @NotNull
+    @Override
+    public List<String> getTabCompletionOptions(
+                                                 @NotNull final MinecraftServer server,
+                                                 @NotNull final ICommandSender sender,
+                                                 @NotNull final String[] args,
+                                                 @Nullable final BlockPos pos)
+    {
+        return new ArrayList<>();
```

Comment:
I think suggesting numbers wouldn't really help the player, may confuse him more.